test/remote/gateways/remote_stripe_payment_intents_test.rb

```diff
@@ -531,6 +531,59 @@ def test_successful_purchase_with_skip_radar_rules
     assert_equal ['all'], purchase.params['charges']['data'][0]['radar_options']['skip_rules']
   end
 
+  # this specific test requires credentials that are setup for AFT
+  def test_successful_authorize_with_aft_data
+    gateway = StripePaymentIntentsGateway.new(fixtures(:stripe_aft))
+    options = {
+      recipient_details: {
+        first_name: 'Jane',
+        last_name: 'Doe',
+        email: 'janedoe@email.com',
+        phone: '+44 7123 456789',
+        address: {
+          country: 'GB',
+          line1: '123 Street',
+          line2: '',
+          postal_code: 'tw11qq',
+          state: 'KA',
+          city: 'Preston'
+        },
+        account_details: {
+          card: {
+            first6: '123456',
+            last4: '1234'
+          },
+          bank: {
+            account_number: '1234567890'
```

Comment:
Hey @yunnydang it occurs to me that if this is an option that customers might be sending to stripe, it's something we would need to scrub from the transcript (and also other places in the transaction 😓 ). I'm going to check with the AM about whether or not this is a need; if not, we may want to rethink leaving this wide open and instead be more prescriptive about how we map the fields 😞